README.md

```diff
@@ -135,6 +155,29 @@ Request timeout is configurable (the default is Node's default of 120 seconds):
 stripe.setTimeout(20000); // in ms (this is 20 seconds)
 ```
 
+Timeout can also be set globally via the config object:
```

Comment:
Ah my understanding was that `stripe.setTimeout` was one of the public methods that we would _not_ be deprecating. As it seems like it might be useful to update the global timeout at some point after initializing.

Reply:
Hmm, I do have vague memories of this conversation, but I can't remember where... Do you remember why we thought that might be useful? Can't think of scenarios where either the config or the method wouldn't be adequate off the top of my head...